x-pack/plugins/observability/public/components/shared/page_template/page_template.tsx

```diff
@@ -53,9 +54,13 @@ export interface ObservabilityPageTemplateDependencies {
   navigationSections$: Observable<NavigationSection[]>;
   getPageTemplateServices: () => KibanaPageTemplateKibanaDependencies;
   guidedOnboardingApi: GuidedOnboardingPluginStart['guidedOnboardingApi'];
+  getChromeStyle$: ChromeStart['getChromeStyle$'];
```

Comment:
same as earlier, I'm guessing these changes should only be applied on the page template components in Observability Shared.